### PR TITLE
Add Password Manager section recommendation, update links section 10 

### PR DESCRIPTION
**Pull request by obutterbach**
- New Password Manager section
- Update link to best practice guidelines in section 10
- Update broken link in section 2

### CODE REVIEW

**Comment by davidread** [comment]
The fixes are great - much valued.

However I'm not sure we need so much on password managers though. Maybe I'm missing something, but weak passwords doesn't seem a huge issue, since our users are required to use 2FA throughout AP. How about a single sentence along with the bit on password policy saying "We recommend you use a password manager to store this and other passwords - e.g. free tier from LastPass", and link to an article. And maybe it's helpful the user if it is placed in the docs at the point they create the password?

**Comment by davidread** [comment]
Ah I see you've added a bit in the Getting Started now - great. I'm just not sure it's useful having an article about password managers in general in the annex.

**Comment by obutterbach** [comment]
Thanks @davidread for your comments, I really appreciate!
Based on our conversation, I would modify this PR

File: 02-start.Rmd [hunk 1]
```diff
@@ -5,7 +5,7 @@
 You use a specially authenticated GitHub account to log into the platform, which you can set up as follows:
 
 ### Sign up to GitHub
-If you do not already have an account, head to https://github.com/ and sign up. When signing up, use all the default options, and make sure you save recovery codes as instructed. It's good practice to choose a username that does not feature upper case characters.
+If you do not already have an account, head to https://github.com/ and sign up. When signing up, use all the default options, and make sure you save recovery codes as instructed. It's good practice to choose a username that does not feature upper case characters. Please use a secure password when signing up following this best practices guideline (see [here](https://github.com/ministryofjustice/itpolicycontent/blob/master/content/security/framework/password-standard.md)). Finally, using a password manager could help you generating strong password - e.g. free tier from LastPass (see [here](https://en.wikipedia.org/wiki/Password_manager) for more information about Password Manager).
```
Comment (davidread) [inline 02-start.Rmd]:
```suggestion
If you do not already have an account, head to https://github.com/ and sign up. When signing up, use all the default options, and make sure you save recovery codes as instructed. It's good practice to choose a username that does not feature upper case characters. Please use a secure password when signing up following this best practices guideline (see [here](https://github.com/ministryofjustice/itpolicycontent/blob/master/content/security/framework/password-standard.md)). Finally, using a password manager could help you generating and recalling a strong password - e.g. free tier from LastPass (see [here](https://en.wikipedia.org/wiki/Password_manager) for more information about password managers).
```

**Comment by obutterbach** [comment]
Good call, I applied your suggestion, if it's green, i would suggest that we merge this branch as it also contains a fix for the bookdown issue

**Comment by joelgsamuel** [comment]
👍 LGTM 🕺

**Pull request merged**